tests/cacl/test_cacl_application.py

```diff
@@ -229,6 +229,29 @@ def parse_int_to_tcp_flags(hex_value):
     return tcp_flags_str
 
 
+def get_token_ranges(separator_line):
+    token_ranges = []
+    start = 0
+    while start != -1:
+        end = separator_line.find(' ', start)
+        if end == -1:
+            token_ranges.append((start, len(separator_line)))
+            break
+        token_ranges.append((start, end))
+        start = separator_line.find('-', end)
```

Comment:
@rawal01 Could you share some more details on logic in this function ? Checking why are we trying to find '-'

A sample string if of format 
u"NTP_ACL  RULE_1         9999  DROP      SRC_IP: 20.0.0.2/32   {'asic0': 'N/A', 'asic1': 'N/A'}"

Reply:
Judy,
 I am using  second line (separator line) to find range for each token 
show acl rule 
Table    Rule    Priority    Action    Match    Status
-------  ------  ----------  --------  -------  --------   <-- Seperator line hence using '-'